### DISCUSSION TITLE
负载均衡load-balance轮询模式是否会导致间歇性断网？

### DISCUSSION THREAD

**Discussion by VaporTang** [vernesong/OpenClash#2948] [Q&A]
如题,最近尝试了下负载均衡模式，然后发现有些问题：
间歇性断网（个人认为不算断网，应该属于类似DNS错误，就比如我在YouTube看视频，然后想去bing，Google或github搜东西（或者说一个我浏览器之前没有开过的新域名）的时候，就要刷新很多次才能进[大概等几分钟]）
恢复连网之后又会不断循环😭
ping的话部分网站ping不通的时候ping ip也ping不通
看网上教程把mwan给移除了，没用
开了meta内核走fake ip tun模式
我感觉可能是负载均衡的问题，但也不敢保证，还在测试没开负载均衡的状态
哪位大佬有思路帮我分析下吗🙏
到时候我会把日志发出来，万分感谢

以及还有一个疑问，AdguardHome到底如何配置才能和openclash高效配合并且防止DNS污染

新年快乐

**Comment by evan618** [comment]
有后续了嘛，大佬

**Reply by VaporTang** [reply]
我后面发现可能有一部分是节点自身的问题，你可以试试换好一点的节点
